src/s3ql/backends/gdrive.py

```diff
@@ -0,0 +1,599 @@
+'''
+backends/gdrive.py - this file is part of S3QL.
+
+Copyright © 2017 Isaac Aymerich <isaac.aymerich@gmail.com>
+
+This work can be distributed under the terms of the GNU GPLv3.
+'''
+
+
+from ..logging import logging # Ensure use of custom logger class
+from ..inherit_docstrings import (copy_ancestor_docstring,prepend_ancestor_docstring, ABCDocstMeta)
+from .common import (AbstractBackend, DanglingStorageURLError, NoSuchObject,retry,retry_generator,AuthenticationError)
+import apiclient.discovery
+import apiclient.http
+import time
+import base64
+import hashlib
+import httplib2
+import ssl
+import http
+import socket
+import io
+import json
+import oauth2client.client
+import tempfile
+
+log = logging.getLogger(__name__)
+logging.getLogger("googleapiclient.discovery").setLevel(logging.WARNING)
```

Comment:
Why is this needed?

Reply:
Because an error of google SDK all the request were shown on the log, impossible to read anything because to much information, this fix the issue avoiding all the requests are writed in the s3ql log

Reply:
And this hasn't been fixed? That's a good example of reasons to avoid the library then :-). If it has been fixed, we should explicitly depend on a fixed version of the module (if we have to use the module at all).